doc/whats_new.rst

```diff
@@ -33,6 +33,8 @@ Changelog
 
     - Add support for the University of Maryland KIT system by `Christian Brodbeck`_
 
+    - Add support for *.elp and *.hsp files to the KIT2FIFF converter and :func:`mne.channels.read_dig_montage` by `Teon Brooks`_ and `Christian Brodbeck`_
```

Comment:
stars render as emphasis, so you either need to escape them or put them in code quotes